### PR TITLE
Documentation for OneStream Connector

### PR DESCRIPTION
**Pull request by os-mik**
This is the documentation for the OneStream connector which is being submitted for certification.

### CODE REVIEW

**Comment by Court72** [comment]
@os-mik,

 Thank you for your contribution. Would you take a moment to sign the [Contributor License Agreement (CLA)](https://github.com/MicrosoftDocs/powerquery-docs/pull/511#issuecomment-1919506092)? After the CLA is signed, someone can review your pull request. 

Thanks!

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Comment by os-mik** [comment]
@microsoft-github-policy-service agree company="OneStream Software"

**Comment by Jak-MS** [comment]
@DougKlopfenstein - 
- Can you review this PR?
- IMPORTANT: When this content is ready to merge, you must add `#sign-off` in a comment or the approval may get overlooked.

NOTE: as a new article, the commits in this PR will need to be moved to the private repo for staging and a full review. Let us know if you'd like our assistance with that.

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Comment by JasonWHowell** [comment]
#assign:ptyx507x
#assign:jonburchel

**Review by DougKlopfenstein — APPROVED** [review]

**Comment by DougKlopfenstein** [comment]
@os-mik 
Could you add a table of contents entry for this connector article? The table of contents file is toc.yml. The connectors are listed in the table of contents in alphabetical order.

**Comment by os-mik** [comment]
@DougKlopfenstein 
Table of Contents updated. Let me know if anything else needed. Thanks!

**Comment by DougKlopfenstein** [comment]
#sign-off

**Comment by DougKlopfenstein** [comment]
Hi @os-mik 
Before this PR can be merged, you'll need to update the pbi-get-data.png image. That Power BI Desktop screen no longer exists. You might need to update your Power BI Desktop to get the modern screen. Also, if you take a new image, make sure any personal information (like a person's name or a server name) is fictional.
#hold-off

**Comment by DougKlopfenstein** [comment]
#sign-off

File: powerquery-docs/connectors/onestream.md [hunk 1]
```diff
@@ -0,0 +1,166 @@
+---
+title: Power Query OneStream connector
+description: Provides support, prerequisites, and instructions on how to connect to your OneStream data.
+author: os-mik
+ms.author: dougklo
+---
```
Comment (Jak-MS) [inline powerquery-docs/connectors/onestream.md:6]:
```suggestion
ms.date: 05/29/2024
ms.topic: conceptual
---
```

**Comment by Jak-MS** [comment]
@os-mik - we'll merge this now, but MS style is for sensitive / private information to be obscured by substituting fictitious content OR by masking with the background color (erasing it). Blurring is not allowed. Can you follow-up with a pull request that replaces the images where you've blurred content? thanks!

cc: @DougKlopfenstein

**Comment by os-mik** [comment]
@Jak-MS thank you, I will submit a new PR next week as I also have to change the logo (company just went through rebranding).
Thanks again!

**Pull request merged**